corehq/apps/users/templates/users/email/deprecated_user.txt

```diff
@@ -0,0 +1,24 @@
+{% load i18n %}
+
+{{ greeting }}
+
+{% blocktrans %}
+As mentioned a week ago, your CommCare user {{ old_username }} has now been migrated to the new user:
+{{ new_username }}.
+
+The following data was copied to your new user:
+ - General user data (name, phone number, email, etc.)
+ - Project space memberships (including roles and locations)
+ - Saved reports
+ - Scheduled reports
+
+The following data was not copied to your new user:
+ - API keys
+ - Pending CommCare invitations
+
+The {{ old_username }} user is no longer active. You can log into your new user with the username
+{{ new_username }}. If you are exempt from SSO, you can follow the Forgot Password workflow to successfully login.
```

Comment:
Is the meaning of "SSO" commonly known? Would it be worth spelling it out rather than using an acronym?

The same also applies to other templates.

Reply:
reminder email text updated after feedback from Erin. This line was scrapped as it's highly unlikely this will apply to any user needing a migration. The users that fall into that category are account admins